src/client/components/homeComponents/HomePosts.jsx

```diff
@@ -0,0 +1,81 @@
+import { useState, useEffect } from "react";
+
+const URL = process.env.REACT_APP_API_URL;
+const postEndpoint = "/post";
+const postURL = URL + postEndpoint;
+
+const HomePosts = () => {
+  const [postArr, setPostArr] = useState([]);
+  const [search, setSearch] = useState("");
+
+  const handleSearch = e => {
+      setSearch(e.target.value);
+  }
+
+  const searchPosts = (searchTerm, postArr) => {
+    if (!searchTerm) {
+      return postArr;
+    }
+    return postArr.filter(post => post.content.toLowerCase().includes(searchTerm) || post.title.toLowerCase().includes(searchTerm) || post.profile.user.username.toLowerCase().includes(searchTerm))
+  }
+
+  const filteredPosts = searchPosts(search, postArr);
+
+  const changeDate = oldDate => {
```

Comment:
what does this do?

Reply:
Changes the DateTime string from SQL into a more readable version.